### PR TITLE
Support higher energy values and change labels to watt hour

### PR DESCRIPTION
**Pull request by madsnedergaard**
## Issue

Part of #4428

## Description

Changes power formatting to handle higher values, and also changes from power (KW) to energy (KWh).

### Double check

- [x] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

### CODE REVIEW

File: web/src/utils/formatting.ts [hunk 1]
```diff
@@ -12,12 +12,21 @@ export const formatPower = function (
...
     return d;
   }
-  const power = `${d3.format(`.${numberDigits}s`)(d * 1e6)}W` //Add a space between the number and the unit
+  const power = `${d3.format(`.${numberDigits}s`)(d * 1e6)}Wh` //Add a space between the number and the unit
     .replace(/([A-Za-z])/, ' $1')
     .trim();
   return power;
 };
 
+export const formatPowerWithSameUnit = function (
```
Comment (tonypls) [inline web/src/utils/formatting.ts:21]: Doesn't look like this one is being used

**Review by tonypls — APPROVED** [review]
LGTM apart from the unused function

File: web/src/utils/formatting.ts [hunk 2]
```diff
@@ -64,21 +73,35 @@ const scalePower = function (maxPower: number | undefined) {
...
 
   if (value < 1e3) {
     return {
-      unit: PowerUnits.MEGAWATTS,
+      unit: EnergyUnits.MEGAWATT_HOURS,
       formattingFactor: 1,
     };
   }
 
+  if (value < 1e6) {
```
Comment (VIKTORVAV99) [inline web/src/utils/formatting.ts:88]: does it make sense to turn this into a switch case now that the if satements have grown?
I feel like they are easier to read over repeated if statements.

File: web/src/utils/formatting.ts [hunk 3]
```diff
@@ -64,21 +73,35 @@ const scalePower = function (maxPower: number | undefined) {
 
   if (value < 1) {
```
Comment (VIKTORVAV99) [inline web/src/utils/formatting.ts:74]: Do we always want to use EnergyUnits or do we want to use PowerUnits for hourly and energy for the rest. For hourly they have the same value and meaning.

**Review by VIKTORVAV99 — APPROVED** [review]
No functional problems so approving. But I left some comments to think about.

File: web/src/utils/formatting.ts (hunk 3 above)
Comment (madsnedergaard) [reply web/src/utils/formatting.ts:74]: I think it makes most sense to stick with energy for consistency :)

File: web/src/utils/formatting.ts (hunk 2 above)
Comment (madsnedergaard) [reply web/src/utils/formatting.ts:88]: Good point, I've rewritten it to be cleaner by using a mapping :)

**Pull request merged**